purview/models/__init__.py

```diff
@@ -81,3 +37,71 @@ def replace(self, node, properties=None):   # pragma: no cover
 
 RelationshipManager.disconnect_all = disconnect_all
 RelationshipManager.replace = replace
+
+
+def story_flow(label):
+    """
+        Get the next/previous node in a story flow/pipeline path
```

Comment:
The docstring contents are indented one too many times. This also needs a period at the end of it.